components/apimgt/org.wso2.carbon.apimgt.gateway/pom.xml

```diff
@@ -351,6 +355,7 @@
                             org.wso2.carbon.context,
                             org.wso2.carbon.mediation.*; version="${carbon.mediation.imp.pkg.version}",
                             org.wso2.carbon.apimgt.tracing.*; version="${carbon.apimgt.imp.pkg.version}",
+                            org.wso2.orbit.re2j.*; version="1.2.wso2v1",
```

Comment:
can't we use the version from the root pom
https://github.com/HiranyaKavishani/carbon-apimgt/blob/e4bd2050598dc79314d4d7d76ada3d1cb50b9bd5/pom.xml#L1656

Reply:
Fixed with 07871ad1b3c4d8f82ef805cbd79ad85375a05c8c